crates/bevy_core_pipeline/src/skybox/mod.rs

```diff
@@ -90,6 +92,7 @@ pub struct Skybox {
     /// After applying this multiplier to the image samples, the resulting values should
     /// be in units of [cd/m^2](https://en.wikipedia.org/wiki/Candela_per_square_metre).
     pub brightness: f32,
+    pub transform: Option<Transform>,
```

Comment:
Do the scale and translation do anything? If not this should probably be a quaternion.

Reply:
It should only require Rotation. I have already adjusted the API to use quaternion, thank you.